[hos-wallpaper] Moves hos-wallpaper to the packages folder

DT3264 opened the pull request:
[hos-wallpaper] Moves hos-wallpaper to the packages folder


While moving hos-wallpaper to the packages folder, it also adds a script to copy the usrroot files and every package that might be needed to copy (such as hos-wallpaper), such script will also allow to easily copy all of those files to an usb drive so it would be easier to both copy and test such changes

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/equetzal/huronOS-build-tools/pull/145).
* #146
* __->__ #145

equetzal commented on base-system/tools/rootcopy.sh:
This file seems to be a helper script different to `hos-wallpaper`, please move it to it's proper PR. It can be at the bottom of this, or a the top. 
Btw, nice tool to have!

DT3264 replied:
Done, added at the top of the stack

DT3264 committed:
[hos-wallpaper] Moves hos-wallpaper to the packages folder


While moving hos-wallpaper to the packages folder, it also adds a script to copy the usrroot files and every package that might be needed to copy (such as hos-wallpaper), such script will also allow to easily copy all of those files to an usb drive so it would be easier to both copy and test such changes

equetzal reviewed (approved):
IWFM
